applications/source/cdc-debezium-source/src/test/java/org/springframework/cloud/stream/app/source/cdc/CdcSourceDatabasesIntegrationTest.java

```diff
@@ -81,8 +85,10 @@ public void mysql() {
 						"--cdc.config.database.port=" + MAPPED_PORT)) {
 			OutputDestination outputDestination = context.getBean(OutputDestination.class);
 			// Using local region here
-			List<Message<?>> messages = receiveAll(outputDestination);
+			List<Message<?>> messages = CdcTestUtils.receiveAll(outputDestination);
 			assertThat(messages).isNotNull();
+			// Message size should correspond to the number of insert statements in the sample inventor DB configured in the debezium/example-mysql:1.9.6.Final:
```

Comment:
Thanks for the comment - this will help when we look back in here in say 6 months and wonder what in the world is 52? Hehe.